Core/HLE/sceUsbCam.cpp

```diff
@@ -153,9 +154,21 @@ static int sceUsbCamSetupVideoEx(u32 paramAddr, u32 workareaAddr, int wasize) {
 
 static int sceUsbCamStartVideo() {
 	std::lock_guard<std::mutex> lock(videoBufferMutex);
-	videoBufferLength = sizeof(sceUsbCamDummyImage);
+
+	int width, height;
+	getCameraResolution(config->type, &width, &height);
+
+	unsigned char* jpegData = nullptr;
+	int jpegLen = 0;
+	__cameraDummyImage(width, height, &jpegData, &jpegLen);
+	videoBufferLength = jpegLen;
 	memset(videoBuffer, 0, VIDEO_BUFFER_SIZE);
-	memcpy(videoBuffer, sceUsbCamDummyImage, sizeof(sceUsbCamDummyImage));
+	memcpy(videoBuffer, jpegData, jpegLen);
+	if (jpegData) {
```

Comment:
Hm, we probably want the memcpy inside here too?

-[Unknown]

Reply:
done